executor/runtime/docker/seccomp/titus.libsonnet

```diff
@@ -64,4 +64,21 @@
       'clone3',
     ],
   },
+  AllowImageBuildingSyscalls: {
+    action: 'SCMP_ACT_ALLOW',
+    names: [
+      'clone',
+      'fsconfig',
+      'fsmount',
+      'fsopen',
+      'fspick',
+      'keyctl',
+      'mount',
+      'move_mount',
+      'pivot_root',
```

Comment:
You are right, keyctl/pivot were added to fuse *only* so we could hack it and piggyback on that profile for image building.

You are hitting on the very reason for a new profile! :)
Lets remove those other syscalls from fuse in another pr